packages/endo/src/zip.js

```diff
@@ -7,9 +7,9 @@ export const readZip = async (data, location) => {
   await zip.loadAsync(data);
   const read = async path => {
     const file = zip.file(path);
-    if (file === undefined) {
+    if (file === undefined || file === null) {
```

Comment:
Would this be simpler?

```suggestion
    if (!file) {
```